libs/components/indicators/src/lib/modules/tokens/tokens.component.ts

```diff
@@ -110,39 +111,39 @@ export class SkyTokensComponent implements OnDestroy {
    * @default true
    */
   @Input()
-  public set focusable(value: boolean) {
-    this._focusable = value;
+  public set focusable(value: boolean | undefined) {
+    this.#_focusable = value !== false;
   }
 
   public get focusable(): boolean {
-    return this._focusable !== false;
+    return this.#_focusable;
   }
 
   /**
    * Specifies an array of tokens to include in the list.
    */
   @Input()
   public set tokens(value: SkyToken[] | undefined) {
-    this._tokens = value;
-    this.tokensChange.emit(this._tokens);
+    this.#_tokensOrDefault = value || [];
+    // The previous behavior was to set `this._tokens = value`, then emit `this._tokens`,
+    // which would emit `undefined` instead of a default value of `[]` returned by the
+    // get accessor when set to `undefined`. Emitting `value` instead of `this.#_tokensOrDefault`
+    // preserves that behavior.
+    this.tokensChange.emit(value);
   }
 
   public get tokens(): SkyToken[] {
-    return this._tokens || [];
+    return this.#_tokensOrDefault;
   }
 
   /**
    * Specifies an observable of `SkyTokensMessage` that can place focus on a
    * particular token or remove the active token from the list.
    */
   @Input()
-  public set messageStream(value: Subject<SkyTokensMessage>) {
-    this._messageStream = value;
-    this.initMessageStream();
-  }
-
-  public get messageStream(): Subject<SkyTokensMessage> {
-    return this._messageStream;
+  public set messageStream(value: Subject<SkyTokensMessage> | undefined) {
+    this.#messageStreamOrDefault = value || new Subject<SkyTokensMessage>();
```

Comment:
Instead of setting a `fooOrDefault` property, why not use a getter for the `messageStream`? If not, it would help me if we specified in our team docs when it's appropriate to use the `fooOrDefault` pattern as opposed to a normal getter.

Reply:
Fixed.